fishtest/fishtest/rundb.py

```diff
@@ -333,7 +333,7 @@ def active_run_lock(self, id):
       self.purge_count = self.purge_count + 1
       if self.purge_count > 100000:
         old = time.time() - 10000
-        self.active_runs = dict((k,v) for k, v in self.active_runs.iteritems() if v['time'] >= old)
+        self.active_runs = dict((k,v) for k, v in list(self.active_runs.items()) if v['time'] >= old)
```

Comment:
I do not think the list () is needed in this context, but it was advised by the 2to3 tool. This code is not performance critical so it does not really matter.